apps/google-analytics-4/frontend/src/components/config-screen/api-access/display/ChecklistUtils.tsx

```diff
@@ -93,113 +95,134 @@ export type ChecklistStatus = {
+        'The Analytics Admin API allows Contentful to fetch the list of properties your service account has access to. Please enable this API inside your Google Cloud project.'
+      ),
       title: CHECKLIST_NAMES.adminApi,
-      description: 'Analytics Admin API must be enabled to use this app',
+      description: 'Analytics Admin API is not enabled',
       disabled: false,
     },
   },
   DataApi: {
     success: {
-      icon: getSuccessIcon('The Analytics Data API is enabled'),
+      icon: getSuccessIcon(''),
       title: CHECKLIST_NAMES.dataApi,
-      description: 'Success!',
+      description: 'Analytics Data API is enabled',
       disabled: false,
     },
     firstTimeSetup: {
-      icon: getArrowIcon('Please enable the Data API to run this check'),
+      icon: getArrowIcon(
+        'The Analytics Data API allows Contentful to fetch analytics data from the property you specify. Please enable this API inside your Google Cloud project to continue.'
+      ),
       title: CHECKLIST_NAMES.dataApi,
       description: 'Analytics Data API is not yet enabled',
       disabled: false,
     },
     invalidServiceAccount: {
       icon: getClockIcon(
-        'Service account errors detected, please install a correctly configured service account installation'
+        'This check will not run until a valid service account key has been provided'
       ),
       title: CHECKLIST_NAMES.dataApi,
-      description: 'Awaiting a correctly configured service account installation',
+      description: `Provide a valid service account key to run this check`,
       disabled: true,
     },
     error: {
-      icon: getErrorIcon('Failed to connect to the Data API - did you enable the api?'),
+      icon: getErrorIcon(
+        'The Analytics Data API allows Contentful to fetch analytics data from the property you specify. Please enable this API inside your Google Cloud project.'
+      ),
       title: CHECKLIST_NAMES.dataApi,
-      description: 'Analytics Data API must be enabled to use this app',
+      description: 'Analytics Data API is not enabled',
       disabled: false,
     },
   },
   GA4Properties: {
     success: {
-      icon: getSuccessIcon(
-        'Your service account has access to assign a Google Analytics 4 property'
-      ),
+      icon: getSuccessIcon(''),
       title: CHECKLIST_NAMES.ga4Properties,
-      description: 'Success!',
+      description: 'Service account has "viewer" access',
       disabled: false,
       checklistUrl: {
         title: 'Details',
         url: 'https://analytics.google.com/analytics/web/',
       },
     },
     firstTimeSetup: {
-      icon: getClockIcon(`Check will run once the ${CHECKLIST_NAMES.adminApi} is enabled`),
+      icon: getArrowIcon(
+        `You need to grant viewer access to your service account in a Google Analytics 4 property.`
+      ),
+      title: CHECKLIST_NAMES.ga4Properties,
+      description: "Service account doesn't have access to any GA4 properties",
```

Comment:
is `GA4` slang? Should we say the full name `Google Analytics 4 properties` here or does it not fit?

Reply:
If it's too long maybe just say `analytics properties`?

Reply:
Yeah agreed. We're short on space in this section so reworded as follows:

>  Service account doesn't have access to a Google Analytics 4 property

Reply:
perfect.